apps/admin/frontend/test/helpers/mock_usb_drive.ts

```diff
@@ -0,0 +1,26 @@
+import { throwIllegalValue } from '@votingworks/basics';
+
+import type { UsbDriveStatus } from '@votingworks/usb-drive';
+
+export function mockUsbDriveStatus(
```

Comment:
Copied from VxScan's `fakeUsbDriveStatus`, which I rename in a later commit. Could move this to `test-utils`.